zips/draft-coinholder-directed-grants.md

```diff
@@ -0,0 +1,192 @@
+**“The Administrator”** refers to an organization that administers the Coinholder Grants Program and manages proposal submission, verification, and reporting processes.  
+
+“ZEC” refers to the native currency of Zcash on Mainnet.  
+
+---
+
+## Abstract
+
+This ZIP defines the **process** for the Coinholder Grants Program established in ZIP 1016.  
+It specifies that the program SHALL operate as a **retroactive grants program**, awarding funds only for work that has been completed and verified.  
+
+The program:  
+* empowers coinholders to directly approve or reject funding proposals;  
+* minimizes administrative overhead by paying only for finished work; and  
+* ensures transparency, accountability, and legal compliance through limited oversight by the Administrator or its delegate and Key-Holder Organizations.  
+
+---
+
+## Motivation
+
+[ZIP 1015](https://zips.z.cash/zip-1015) introduced a temporary, non-direct funding model that redirected 12% of block rewards to a protocol lockbox and 8% to Zcash Community Grants (ZCG).  
+
+[ZIP 1016](https://zips.z.cash/zip-1016) extends this by defining the **Community and Coinholder Funding Model**, allocating 12% of block rewards to a Coinholder Grants Program and 8% to ZCG through Zcash’s third halving.  
+
+While ZIP 1016 defines the funding structure and governance boundaries, it does not prescribe an implementation process for coinholder-directed grants. This ZIP provides that missing specification.  
+
+A **retroactive grants** model aligns with Zcash’s principles of accountability and decentralization by:  
+* rewarding completed, verifiable work rather than speculative milestones;  
+* reducing administrative complexity for Key-Holder Organizations;  
+* enabling direct, transparent coinholder participation; and  
+* ensuring that funds are only disbursed for measurable contributions that advance Zcash.  
+
+---
+
+## Requirements
+
+1. **Retroactive Funding Model** — Grants MUST be awarded solely for work that is demonstrably complete and publicly verifiable.  
+2. **Coinholder Voting Process** — All grants MUST be approved by a coinholder vote meeting the participation threshold defined in ZIP 1016 (≥ 420,000 ZEC voted).  
+3. **Transparency and Reporting** — Proposal information, vote outcomes, and disbursements MUST be published publicly by the Administrator or its delegate.  
+4. **Administrative Safeguards** — The Administrator or its delegate SHALL manage proposal submission, eligibility screening, compliance checks, and quarterly reporting, including KYC for grants > 50,000 USD equivalent.  
+5. **Keyholder Oversight** — Key-Holder Organizations MUST execute approved disbursements unless vetoed under the conditions defined in ZIP 1016 (Veto Process).  
+6. **Legal Compliance** — All parties MUST comply with applicable legal, tax, and reporting obligations.  
+7. **Security and Custody** — Multisig keys MUST be secured per the standards in ZIP 1016.  
```

Comment:
This content should be in `## Specification`.

The Requirements section is explicitly to define what requirements _this ZIP_ must satisfy; i.e. constraints on this specification itself, or on the author of the specification. The Requirements section should not impose any conformance requirements. The requirements section guides the authoring of the specification; it is not specification itself.